README.md

```diff
@@ -48,3 +48,42 @@ Because Markdown files rendered by GitHub Pages should behave similar to Markdow
   gems:
     - jekyll-relative-links
   ```
+
+## Configuration
+
+You can configure this plugin in `_config.yml` under the `relative_links` key. This is optional and defaults to:
+
+```yml
+relative_links:
+  enabled:     true
+  collections: false
+```
+
+### Processing Collections
+
+Setting the `collections` option to `true` enables relative links to and from collection items (including posts).
+
+Assuming this structure:
+
+~~~
+├── _posts
+│   └── 2016-01-01-test.md
+├── _config.yml
+├── index.md
+└── some-category
+    └── _posts
+        └── 2016-01-01-test.md
+~~~
```

Comment:
Maybe add a `_my_collection` folder with a `some_doc.md` file to make these examples match the official collections documentation from https://jekyllrb.com/docs/collections/ here?

(At first I was confused if this is some special kind of _posts and not a real collection how I know them)